src/screens/news-feed-screen/news-feed-screen.spec.js

```diff
@@ -0,0 +1,111 @@
+import {newsFeedScreenDriver} from './news-feed-screen.driver';
+import {Linking} from 'react-native';
+import {UNKNOWN} from '../../strings';
+import dateFormat from 'dateformat';
+import Chance from 'chance';
+
+const chance = Chance();
+
+describe('News Feed Screen', () => {
+  let driver;
+
+  beforeEach(() => {
+    driver = newsFeedScreenDriver();
+  });
+
+  const defaultProps = {
+    fetchCovidNewsAction: () => {},
+    articles: [],
+  };
+
+  it('renders an empty container when articles could not be fetched', () => {
+    driver.setProps(defaultProps);
+
+    expect(driver.container()).toBeDefined();
+    expect(driver.articles().length).toEqual(0);
+  });
+
+  it('renders a single article along with all data associated correctly', () => {
+    const article = buildArticle();
+    const articles = [article];
+    const expectedTitle = article.title;
+    const expectedSourceName = article.source.name;
+    const expectedDate = dateFormat(article.publishedAt);
+    const expectedImageUrl = article.urlToImage ?? undefined;
+    driver.setProps({...defaultProps, articles});
+
+    expect(driver.articles().length).toEqual(1);
+    expect(driver.articleAtIndex(0).title()).toEqual(expectedTitle);
+    expect(driver.articleAtIndex(0).sourceName()).toEqual(expectedSourceName);
+    expect(driver.articleAtIndex(0).timestamp()).toEqual(expectedDate);
+    expect(driver.articleAtIndex(0).imageUri()).toEqual(expectedImageUrl);
+  });
+
+  it('renders multiple different articles retrieved in order', () => {
+    const articles = Array.from(
+      {length: chance.natural({min: 2, max: 10})},
+      () => buildArticle(),
```

Comment:
if your generator function does not have any arguments, you just use it's ref (buildArticle), instead of creating anonymous (function () => buildArticle)

Reply:
you can also write this generator using chance, i.e. 
```
chance.unique(buildArticle, chance.integer({min: 2, max: 10})),
```